modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/analytics/authentication/AnalyticsLoginDataPublishHandlingTestCase.java

```diff
@@ -0,0 +1,809 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Can't we extend the existing AnalyticsLoginTestCase without duplicating all stuff here ?

Reply:
done